appveyor.yml

```diff
@@ -3,7 +3,6 @@ environment:
   APPVEYOR_SAVE_CACHE_ON_ERROR: true
   matrix:
     - MSYSTEM: MINGW64
-    - MSYSTEM: MINGW32
```

Comment:
Build must also be compiled on 32-bit systems.

Reply:
There are libmng issues with 32-bit systems.

Reply:
You can compile without requiring libmng as autotools does.

Reply:
Also this problem should be fixed by this commit (https://github.com/synfig/synfig/commit/f8f918f6610ed01f66dc1d89b79a17b53123f8cb)

Reply:
This part is fixed, it builds on 32 bit system as you can see [here](https://ci.appveyor.com/project/Synfig/synfig/builds/34234842).